packages/core/crates/core-misc/src/environment.rs

```diff
@@ -157,6 +160,8 @@ impl PackageJsonFile {
 }
 
 #[derive(Serialize, Clone)]
+#[cfg(feature = "wasm")]
```

Comment:
This line should be probably removed, since it is using the `wasm_bindgen_if` macro already.